includes/class-main.php

```
@@ -243,6 +264,29 @@ private function setup_hooks(): void {
+		}
+
+		// Only run this check once per hour to avoid constant I/O.
+		if ( get_transient( 'wppo_wp_cache_fix_checked' ) ) {
+			return;
+		}
+
+		set_transient( 'wppo_wp_cache_fix_checked', 1, HOUR_IN_SECONDS );
+
+		require_once WPPO_PLUGIN_PATH . 'includes/class-activate.php';
+		Activate::add_wp_cache_constant();
+	}
```

PHPMD (2.15.0)

[error] The method maybe_fix_wp_cache is not named in camelCase. (undefined)

(CamelCaseMethodName)